main.tf

```diff
@@ -28,7 +28,7 @@ locals {
 }
 
 resource "ibm_scc_provider_type_instance" "scc_provider_type_instance_instance" {
```

Comment:
can you leave a code comment above this resource block to explain what its doing (aka attaching an SCC Workload Protection instance)